kubernetes/istio-rbac/istio-rbac.yaml

```diff
@@ -97,7 +97,22 @@ metadata:
   name: istio-sidecar-role-binding
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: details-service-account
```

Comment:
I agree that we should make istio-auth-X.yaml and istio-rbac.yaml generic configurations, not specific to bookinfo. We are referring to these files in our general tutorial pages.
I think setting service accounts and bindings for bookinfo applications shall be moved out to bookinfo.yaml, if possible.